Simplenote/TagListViewController.swift

```diff
@@ -376,89 +372,104 @@ private extension TagListViewController {
         cell.hasClearBackground = true
         cell.imageTintColor = .simplenoteTintColor
     }
+}
 
-    func shouldSelectCell(at indexPath: IndexPath) -> Bool {
-        guard let section = Section(rawValue: indexPath.section) else {
-            return false
+// MARK: - Selection
+//
+private extension TagListViewController {
+    func updateCurrentSelection() {
+        tableView.selectRow(at: currentSelection, animated: false, scrollPosition: .none)
+    }
+
+    var currentSelection: IndexPath? {
+        let filter = NotesListFilter(selectedTag: SPAppDelegate.shared().selectedTag)
```

Comment:
I'm wondering if the TagListViewController should actually own the Selected Tag.

(And down the road, change its type from String to NotesListFilter?)

Reply:
That's a good point. I think there are several options, we can also have a separate object holding the current state of the app and inject it when needed. 
Note list for example can listen for changes in that object and update the list.